packages/app/src/components/__tests__/App.spec.js

```diff
@@ -137,9 +138,10 @@ describe('App', () => {
                 ).toBeCalledTimes(1)
 
                 expect(props.addParentGraphMap).toBeCalledTimes(1)
-                expect(props.setCurrentFromUi).toBeCalledTimes(1)
```

Comment:
Shouldn't `setCurrentFromUi` still be called? Only `setVisualization` was removed, right?

Reply:
Yes, the test is a couple of lines below.